crates/ra_proc_macro_srv/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+edition = "2018"
+name = "ra_proc_macro_srv"
+version = "0.1.0"
+authors = ["rust-analyzer developers"]
+publish = false
+
+[lib]
+doctest = false
+
+[dependencies]
+ra_tt = { path = "../ra_tt" }
+ra_mbe = { path = "../ra_mbe" }
```

Comment:
@matklad  We need mbe here for parsing string to `ra_tt::Token` which is used for lib_proc_macro bridge api.